recipe/meta.yaml

```diff
@@ -21,21 +21,29 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
+    - poetry-core
 
   run:
-    - python >=3.6
-    - snowballstemmer
+    - python
+    - snowballstemmer >=2.2.0
+    - tomli >=1.2.3
```

Comment:
Must be `tomli >=1.2.3  # [py<311]`, see https://github.com/PyCQA/pydocstyle/blob/07f6707e2c5612960347f7c00125620457f490a7/pyproject.toml#L24

Reply:
If we will build for python 3.7 then add:
`    - importlib-metadata >=2.0.0,<5.0.0  # [py<38]`
see https://github.com/PyCQA/pydocstyle/blob/07f6707e2c5612960347f7c00125620457f490a7/pyproject.toml#L25

Reply:
Thanks!